README.md

```diff
@@ -1,2 +1,24 @@
-# DeveloperGuides
-A repository for organizing style guides, templates, tools, and techniques for UC San Diego's Development and Web Services Team.
+# Developer Guides
+A repository for organizing style guides, templates, tools, and techniques for the UC San Diego Development and Web Services Team.
+
+* [Protocol](protocol/README.md)
+  * [Code Review](protocol/code_review.md)
+  * [Git](protocol/git.md)
+  * [Release Management](protocol/release_management.md)
+* [Templates](templates/README.md)
+  * [Code Climate](templates/.codeclimate.yml)
+  * [Copyright](templates/UC_Copyright_Notice.txt)
+* [Best Practices](best-practices/README.md)
+  * [Deployment](best-practices/deployment.md)
+  * [Documentation](best-practices/documentation.md)
+  * [Project README.md](best-practices/project_readme.md)
+
+## Contributing
+Since this repository covers best practice recommendations for the entire team,
+no PR will be merged until the entire team gives a :thumbsup:
```

Comment:
I should probably clarify that I mean for this as a contribution requirement for this guides repository, since everyone is impacted by changes. Not for all of our repositories. Clearly I should use better wording to clarify that.